src/types/libdef/npm-custom/workbox-window_vx.x.x.js

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+declare module 'workbox-window' {
```

Comment:
initially I kept the placeholder, but then I found a bug that could have been avoided if the libdef was correct, so I decided to implement it after all.